Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2_test.py

```diff
@@ -65,15 +65,14 @@ def test_ip_command(requests_mock):
     ip = '1.1.1.1'
 
     mock_response = util_load_json('test_data/ip_1.1.1.1_response.json')
-    requests_mock.get(f'https://ipinfo.io/{ip}/json', json=mock_response)
-
     client = Client(api_key='',
                     base_url='https://ipinfo.io',
                     verify_certificate=False,
                     proxy=False,
                     reliability=DBotScoreReliability.C)
 
-    command_results = ipinfo_ip_command(client, ip)
-    parsed_context = [command_result.to_context() for command_result in command_results]
+    mocker.patch.object(client, 'http_request', return_value=mock_response)
+    command_results = mocker.patch('ipinfo_v2.CommandResults')
+    ipinfo_ip_command(client, ip)
 
-    assert "lat|lng" in parsed_context[1].get("HumanReadable")
+    assert "lat|lng" in command_results.call_args[1].get("readable_output")
```

Comment:
That's what you're looking for in the test?

Reply:
yes, my change includes those two that were missing from the table, and appeared alone in the info part
![Screen Shot 2022-01-26 at 16 37 29](https://user-images.githubusercontent.com/93524335/151183067-d993f476-6fc3-4d22-92de-954442239754.png)

Reply:
But this happened before. Don't you want to check that the new output is the right one?

Reply:
![Screen Shot 2022-01-26 at 17 06 23](https://user-images.githubusercontent.com/93524335/151188913-7088c45d-b6b9-48b3-9def-8eb5acd61a03.png)

this is the new one

Reply:
cool. Is that all the difference? if not, might want to check if the ip/company in it as example.